src/es/catharsisworld/src/eu/kanade/tachiyomi/extension/es/catharsisworld/CatharsisWorld.kt

```diff
@@ -51,6 +58,42 @@ class CatharsisWorld : Madara(
         setUrlWithoutDomain(element.selectFirst("a")!!.attr("href"))
     }
 
+    override fun pageListParse(document: Document): List<Page> {
```

Comment:
Have not looked at the Madara class, but if this is an update/improvement to the current implementation it should be moved there.

If the difference is site specific (selector, variable names) it should be extracted into some overridable variable. If simple enough you might not need to bump Madara.

Reply:
> Have not looked at the Madara class, but if this is an update/improvement to the current implementation it should be moved there.
> 
> If the difference is site specific (selector, variable names) it should be extracted into some overridable variable. If simple enough you might not need to bump Madara.

Theme's is name "Kozaku" according to wordpress scanners.
In case you are curious https://github.com/manga-download/haruneko/blob/997739b68b0a45ead51b2e3dab8476cb43a287e5/web/src/engine/websites/templates/Kozaku.ts#L30